kadi.cfg

```diff
@@ -0,0 +1,30 @@
+[commands]
+# Default lookback for previous approved loads (days).
+# default_lookback = 30
```

Comment:
Why is it the defaults are also commented out in the config?

Reply:
That's pretty much standard practice for auto-generated config files. See e.g. any `conf.py` in the sphinx docs.